integrations/docker/images/chip-build-telink/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN set -x \
     && : # last line
 
 # Setup Zephyr
-ARG ZEPHYR_REVISION=295b4e6dedc4f40701962e0ed0be1090be5c0eee
+ARG ZEPHYR_REVISION=dac17979d626bb820268660b4272a56d04d8083b
```

Comment:
This does not work: we need dockerfiles to be a fixed version that is always identican when built.

If we change this, we will make a telink docker that may have different content depending on what version was built.

If you want, you could delete the docker file from here and add a README pointing to what docker file needs to be used. Or use some different numbering scheme (but then docker rebuilds will be needed).

Unfortunately we never had this situation (a branch needing a different docker) so we do not have a clear way to handle it. Maybe just changing version in `.github/workflows` also works.

Reply:
I mean that in .github/workflows/examples-telink.yaml we already changed telink docker image version for 1.0-branch previously. 
And connectedhomeip/chip-build-telink:0.6.06 image which is active for Telibk 1.0-branch is builded from Zephyr hash dac17979d626bb820268660b4272a56d04d8083b.
Zephyr hash 295b4e6dedc4f40701962e0ed0be1090be5c0eee - is pretty old one and compatible with Telink commit under release tag, but not with latest commit in 1.0-branch.
I make misunderstanding for our customers when they use 1.0-branch.
But its you choose, it this is not allowed, I will revert this change .

Reply:
ok, Have I revert this change in docker file? This change is not mandatory for us, just as pointer of hash into corresponding docker which used in actual commit.

Reply:
Yes, please revert this change. The rest should be good to go.

Reply:
Done